packages/frontend/src/creation-form/components/collaterals/index.tsx

```diff
@@ -129,7 +129,7 @@ export const Collaterals = ({
                     }),
                     ...cachedTokenInfoWithBalanceInChain(CCT_CHAIN_ID),
                 ],
-            };
+            } as TokenListWithBalance;
```

Comment:
Are you sure we need this?

Reply:
No, we don't actually, TS was complaining but I guess it was a temporary issue; fixed in 4f3b14b42e9e82880db9f37455415784afe91c42.